db/migrate/20150221033750_drop_reservation_listings_table.rb

```diff
@@ -0,0 +1,5 @@
+class DropReservationListingsTable < ActiveRecord::Migration
+  def change
+    drop_table :reservation_listings
+  end
```

Comment:
Yea I think that a listing should have many reservations and a reservation belongs to a listing, with that relationship we don't need the join table